Post play test lvl mods

JonlineRB opened the pull request:
Includes the requested modifications to lvls according to playtest feedback.

Dialogue updates and a better explanation of the charge block.

Increased timer of stone block to 2s

JonlineRB committed:
stoneblock change, lvl 2 adjusted

JonlineRB committed:
Lvl 5 modified

JonlineRB committed:
lvl 12 modification

After testing, it is very difficult to solve this level without using chain mechanic (i didn't succeed).

Difficulty should still be manageable

JonlineRB committed:
changed the lvls order

17 -> 16; 18 -> 17; 16->18

JonlineRB committed:
Dialogues update

Accomodates lvl change, tutorial for charge block

JonlineRB committed:
more dialogue updates

lee17017 commented on Assets/Resources/Texts/StoryDialogues.txt:
```diff
@@ -139,32 +139,38 @@ Balagan: Like love?
+Balagan: Yes?
+Da'at: You will kind of... Explode?
+Balagan: WHAT?!
+Da'at: Don't worry, I'm sure you will be fine. Any blocks caught in the explosion though will be anahilated on the spot. Have fun!
```
annihilated

lee17017 reviewed (approved):
Maybe cut the lvl2 length in half, it feels too long for lvl 2

JonlineRB committed:
Level 2 modified

It's way shorter and simpler.
Still wood only, meant to introduce players to the platforming, controls and physics

JonlineRB committed:
fixed dialogue error
